src/client/app/components/dashboard/dashboard.component.ts

```diff
@@ -1,5 +1,5 @@
 import {Component, OnInit} from '@angular/core';
```

Comment:
Try not to commit something you didn't really modify. Clouds up the amount of code in the PR.